sdk/template-LLC/Azure.Template.LLC/src/swagger/swagger.json

```diff
@@ -0,0 +1,123 @@
+﻿{
+  "swagger": "2.0",
```

Comment:
Would it be possible to make these operations look like real service operations?  We've been using PetStore as toy examples elsewhere.  I think it would help service teams understand what's happening a little better.

Reply:
I tried to add the simplest version of swagger. Do you have any swagger on your mind which I can use here as an example?

Reply:
Can we use any service swagger directly and keep only 2-3 methods? This way we will have real service operations and we will be able to add live tests?

Reply:
Discussed offline - Added CRUD operations in the swagger.